.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.yarnpkg.com/
```

Comment:
can this be removed?

Reply:
Yup. I was trying to handle work from my work laptop and it was using our internal registry. I will remove this.